addons/website/static/tests/tours/snippet_image_gallery.js

```diff
@@ -0,0 +1,38 @@
+odoo.define("website.tour.snippet_image_gallery", function (require) {
+"use strict";
+
+const tour = require("web_tour.tour");
+const wTourUtils = require("website.tour_utils");
+
+tour.register("snippet_image_gallery", {
+    test: true,
+    url: "/",
+}, [{
+    trigger: "a[data-action=edit]",
+    content: "Click on the Edit button.",
+    extra_trigger: ".homepage",
+    },
+    wTourUtils.dragNDrop({
+        id: "s_image_gallery",
+        name: "Image Gallery",
+}), {
+    content: "Click on the first new image",
+    trigger: ".o_select_media_dialog img[title='s_default_image.jpg']",
+}, {
+    content: "Click on the second new image",
+    trigger: ".o_select_media_dialog img[title='s_default_image2.jpg']",
+}, {
+    content: "Click on Add",
+    trigger: "button:has(span:contains('Add'))",
+}, {
+    content: "Click on the image of the Image Gallery snippet",
+    trigger: ".carousel img",
+}, {
+    content: "Click on Remove Block",
+    trigger: ".o_we_customize_panel we-button[title='Remove Block']",
```

Comment:
This step should probably make sure that it targets the Image Gallery's Remove block button. If I'm not wrong, this step could also apply to a single image that we selected right? 🤔